Source/Core/DolphinWX/UINeedsControllerState.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <atomic>
+
+static std::atomic<bool> s_needs_controller_state{false};
```

Comment:
Just FYI, Common::Flag can be used for this. Not sure if it's worth changing std::atomic<bool> to it though.